projects/resource-tags.md

```diff
@@ -0,0 +1,82 @@
+
+This Working Group proposes to define semantic conventions for a set of commonly used resource tags, such as environment, application, owner, and data_sensitivity etc and incorporate them into the OpenTelemetry specification. Standardizing the meaning of these tags will allow telemetry pipelines to treat them as first-class, interoperable metadata. For example, resource detectors in OpenTelemetry can surface these tags from platform metadata, making them accessible to downstream tools in a consistent format. With shared semantics in place, observability and security platforms can enable features like governance-aware dashboards, suggest default security postures, cross-cloud resource attribution etc without relying on cloud- or vendor-specific integrations.
+
+---
+
+### Current Challenges
+
+- **No shared semantics**: Common tags like `environment`, `owner`, or `cost_center` are widely used but lack standardized meaning, leading to inconsistent usage and interpretation.
+
+- **Cross-cloud fragmentation**: Tags are implemented differently across providers, making it hard to reason about resources in a consistent way across clouds and tools.
+
+- **Limited utility in observability**: Tags show up in telemetry, but their ambiguous structure makes them hard to use for dashboards, alerting, or policy logic.
+
+---
+
+### Goals, objectives, and requirements
+
+The goal of this project is to define a shared understanding for a set of commonly used resource tags, such as `environment`, `owner`, `cost_center` etc. These definitions will make it easier for OpenTelemetry and related tools to interpret and act on tag metadata consistently—supporting better automation, governance, and observability across environments. Initial areas of work include:
+
+* [Define the initial set of standardized tags and their recommended structure or values]
+* [Ensure alignment with existing OpenTelemetry resource attributes where applicable]
+* [Collaborate with the open source community and other cloud providers for cross-domain alignment]
+
+---
+## Deliverables
+Initial deliverables will include:
+
+* A specification defining common resource tags such as `environment`, `application`, `owner`, `cost_center`, `business_unit`, `data_sensitivity` etc
+* Recommended value sets and usage guidance for applicable tags 
+* Alignment with existing OpenTelemetry resource attributes where relevant
+* Documentation and examples for adoption by cloud providers, observability platforms, and security tools
+
+As adoption grows, we expect to expand this set iteratively based on feedback from users and collaborators across the ecosystem.
+
+---
+## Staffing / Help Wanted
+
+We are seeking domain experts to work on the definition, alignment, and adoption of standardized resource tags across cloud platforms and observability systems.
```

Comment:
please add me as a contributor, I can help with aligning this work with the existing semantic conventions